### PR TITLE
Add markdown/richtext support in the app

### LINKED ISSUE #143: Add markdown/richtext support in the app (we have markdown fields in CMS)
Currently, in the CMS we have an option to enter text with a markdown widget but it is not supported in the code, it will be interpreted as a regular string.

### PR DESCRIPTION
**Pull request by celinawierzbicka**
Closes #143

### CODE REVIEW

File: content/about-me/index-1.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-aboutPageTitle:
-aboutPageDescription:
-aboutPageImage:
+aboutPageTitle: null
```
Comment (mecies) [inline content/about-me/index-1.md]: Why was metadata added?

File: content/resume/index-1.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-resumePageTitle:
-resumePageDescription:
-resumePageImage:
+resumePageTitle: Title
```
Comment (mecies) [inline content/resume/index-1.md]: Why was metadata added?

File: gatsby-node.js [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+
+const convertMarkdown = (markdown) => {
+  return remark().use(remarkHTML).processSync(markdown).toString();
+};
+
+exports.onCreateNode = ({ node }) => {
+  if (node.frontmatter) {
+    if (node.frontmatter.description) {
+      const markdown = node.frontmatter.description;
+      // eslint-disable-next-line no-param-reassign
```
Comment (mecies) [inline gatsby-node.js]: We can add it to global project rules because it occurs frequently

**Review by mecies — CHANGES_REQUESTED** [review]
Good job 👏 
Some minor changes and questions 😄

File: content/about-me/index-1.md (hunk 1 above)
Comment (celinawierzbicka) [reply content/about-me/index-1.md]: I think this is a leftover after yous pushes to this branch, I will remove it

Comment (mecies) [reply content/about-me/index-1.md]: Haha true, please remove ❤️ 
It was a test whether CMS worked

**Review by mecies — CHANGES_REQUESTED** [review]
In `.eslintrc.js` add     `'no-param-reassign': 0`, to rules

**Review by mecies — APPROVED** [review]
Great job ✅ 👏

**Pull request merged**